recipe/meta.yaml

```diff
@@ -16,23 +16,25 @@ build:
     - proxystore-relay = proxystore.p2p.relay.run:cli
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 0
+  number: 1
 
 requirements:
   host:
-    - python >=3.8
+    - python {{ python_min }}
     - setuptools >=64.0
     - setuptools-scm
     - pip
   run:
-    - pydantic >=2
-    - tomli-w
-    - python >=3.8
+    - python >={{ python_min }}
     - click !=8.1.4
     - cloudpickle >=1.6.0
     - cryptography >=39.0.1
     - globus-sdk >=3.3.0
     - lazy-object-proxy >=1.6.0
+    - pydantic >=2
+    # for all python, not just py<3.11, to stay noarch: python
+    - tomli
```

Comment:
grayskull doesn't know how to handle this kind of a case properly.  It isn't fully capable of handling `pyproject.toml` files and it misses packages that are only for some python versions.  Because of this, I'm disabling automerge so we (I) can check dependencies for each update to make sure grayskull hasn't missed important changes.

Reply:
If I understand the changes correctly, then the primary change is `python >={{ python_min }}` which causes the built infrastructure to use Python 3.9 rather than 3.12. We used `pip check` before and they passed without any issue as the dependencies were not required for Python 3.12. Still this seems to be a general decision https://github.com/conda-forge/cfep/blob/main/cfep-25.md

Reply:
Right, the problem is that `pip check` was being run on the maximum (rather than the minimum) supported python in general and was causing trouble.  Missing dependencies like this one are exactly what that CFEP is meant to catch.